src/test/java/org/fundacionjala/coding/abel/FizzBuzzTest.java

```diff
@@ -0,0 +1,53 @@
+package org.fundacionjala.coding.abel;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ * this class make to test a the FizzBuzz class.
+ */
+public class FizzBuzzTest {
```

Comment:
Could you increase the code coverage ?
https://codecov.io/gh/AT-08/coding/pull/67/diff?src=pr&el=tree#diff-c3JjL21haW4vamF2YS9vcmcvZnVuZGFjaW9uamFsYS9jb2RpbmcvYWJlbC9GaXp6QnV6ei5qYXZh